gnomad/utils/annotations.py

```diff
@@ -509,6 +524,28 @@ def annotate_freq(
         + sample_group_filters
     )
 
+    # Add additional groupings to strata, e.g. strata-pop, strata-sex, strata-pop-sex
+    if additional_strata_grouping_expr is not None:
+        for strata in additional_strata_grouping_expr:
+            if strata not in cut_data.keys():
+                raise KeyError(
+                    "%s is not an existing annotation and thus cannot be combined with"
+                    " additional strata"
+                )
```

Comment:
Will this ever happen? strata should be in cut_data right because it's added to `_freq_meta_expr` which is annotated as `mt._freq_meta` and that is used to build the cut_dict for the cut_data aggregation

Reply:
You're right it should not! This is an artifact of a previous approach, before we added `additional_strata_groupings_expr` to _freq_meta_expr- will remove this check.